# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/moz.build
# Commit: 99b2e5c55a25
# Full Hash: 99b2e5c55a25f47f9a5a0650607cd54ce966c28b
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-05 16:09:03
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add D3D11 RenderCompositor for SWGL. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95797
# ==============================================================================

diff -r 96087b511fe9 -r 99b2e5c55a25 gfx/webrender_bindings/moz.build
--- a/gfx/webrender_bindings/moz.build	Thu Nov 05 00:51:24 2020 +0000
+++ b/gfx/webrender_bindings/moz.build	Thu Nov 05 09:14:52 2020 +0000
@@ -11,6 +11,7 @@
     "RenderBufferTextureHost.h",
     "RenderBufferTextureHostSWGL.h",
     "RenderCompositor.h",
+    "RenderCompositorD3D11SWGL.h",
     "RenderCompositorEGL.h",
     "RenderCompositorOGL.h",
     "RenderCompositorSWGL.h",
@@ -34,6 +35,7 @@
     "RenderBufferTextureHost.cpp",
     "RenderBufferTextureHostSWGL.cpp",
     "RenderCompositor.cpp",
+    "RenderCompositorD3D11SWGL.cpp",
     "RenderCompositorEGL.cpp",
     "RenderCompositorOGL.cpp",
     "RenderCompositorSWGL.cpp",
@@ -118,3 +120,4 @@
 FINAL_LIBRARY = "xul"
 
 CXXFLAGS += CONFIG["TK_CFLAGS"]
+CXXFLAGS += CONFIG["MOZ_CAIRO_CFLAGS"] 